### ISSUE TITLE
I wonder how hard it would be to have a boni system for crafting

### ISSUE THREAD

**Issue by spiderbit** [stulu08/Medieval-Dynasty-Mod-Loader#5]
So basically I think a lot of fun in crafting besides making money is about having different degrees of success, and then wear maybe a special version with some boni.

From your tutorials and the links it was not obvious to me how to do that (even I am not sure I can even use this tools inside of linux).

Would that be a overwrite mod? Is there the code on how things spawn from crafting and can you change that not as fixed value but as a function to calculate the armor or damage value? Partially random partially skill level.

**Comment by stulu08** [comment]
Do you mean a system like in Skyrim? Where your end product can be better or worse depending on your skill, or in this case just random?
I assume it is possible, but would require a core or blueprint mod. You would need to look into the weapon and armor system and see if the actual values are fixed or not. If they are fixed, it would not be easy to implement such a system, since you would need to keep track of every item and overwrite the stats. If these values are assigned on generation of the item (spawning, crafting, …) it would be easier since you would just need to overwrite the function generating these stats.

I will try to look into this.

**Comment by spiderbit** [comment]
Yes depending on skill level but also random so depending on your skill your range of results get better.
